Snowflake.Data/Core/FileTransfer/StorageClient/SFGCSClient.cs

```diff
@@ -343,32 +291,18 @@ public async Task UploadFileAsync(SFFileMetadata fileMetadata, Stream fileByteSt
             catch (WebException ex)
             {
                 fileMetadata = HandleUploadFileErr(ex, fileMetadata);
```

Comment:
log here is missing

Reply:
Added the logging in `HandleUploadFileErr`